core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -543,17 +508,54 @@ void updateGroupListOfUser(String userID, String[] deletedGroupIDs, String[] new
      */
     Group renameGroup(String groupID, String newGroupName) throws UserStoreException;
 
+    // User centric group related implementation methods ..............................................................
```

Comment:
Please consider the same comments in https://github.com/wso2/carbon-kernel/pull/2580/files#r375075330